detox/src/utils/fsext.js

```diff
@@ -6,8 +6,14 @@ async function getDirectories (rootPath) {
   let dirs = [];
   for (let file of files) {
     let pathString = path.resolve(rootPath, file);
-    if ((await fs.lstat(pathString)).isDirectory()) {
+    const fileStats = await fs.lstat(pathString);
```

Comment:
Any reason not to change the original line to
```js
if ((await fs.stat(pathString)).isDirectory()) {
```
?

As I understand it, `stat` follows symbolic links, while `lstat` returns information about the link itself.

From the documentation:
https://nodejs.org/api/fs.html#fs_fs_lstat_path_options_callback

> `lstat()` is identical to `stat()`, except that if path is a symbolic link, then the link itself is stat-ed, not the file that it refers to.

Reply:
No reason, I just didn't see it in the docs. I will change it to `fs.stat`.